plugin/src/test/java/io/jenkins/plugins/forensics/git/miner/GitMinerStepITest.java

```diff
@@ -58,6 +58,12 @@ public void shouldMineRepositoryIncrementally() throws IOException {
         build = buildSuccessfully(job);
 
         getJenkins().assertLogContains("created report for 0 files", build);
+
+        writeFileAsAuthorFoo("Second");
+
+        build = buildSuccessfully(job);
+
+        getJenkins().assertLogContains("created report for 2 files", build);
```

Comment:
Shouldn't this be 1 actually? You only changed one file...

Reply:
It should, but the commit always contains all files, even if they were not changed.  [See this answer](https://stackoverflow.com/a/40609231).  I could change the log, so it only shows the number of changed files.

Reply:
Ah, right, I forgot about that. No then let it this way...